assets/src/data/model/datetime/status-helper.js

```diff
@@ -86,25 +86,25 @@ export const isTrashed = ( DateTimeEntity ) => {
 	return DateTimeEntity.deleted;
 };
 
-/**
- * @function
- * @param {Object} DateTimeEntity model object
- * @return {boolean} true if date is cancelled
- */
-export const isCancelled = ( DateTimeEntity ) => {
-	assertDateTimeEntity( DateTimeEntity );
-	return DateTimeEntity.cancelled;
-};
-
-/**
- * @function
- * @param {Object} DateTimeEntity model object
- * @return {boolean} true if date has been postponed
- */
-export const isPostponed = ( DateTimeEntity ) => {
-	assertDateTimeEntity( DateTimeEntity );
-	return DateTimeEntity.postponed;
-};
+// /**
+//  * @function
+//  * @param {Object} DateTimeEntity model object
+//  * @return {boolean} true if date is cancelled
+//  */
+// export const isCancelled = ( DateTimeEntity ) => {
+// 	assertDateTimeEntity( DateTimeEntity );
+// 	return DateTimeEntity.cancelled;
+// };
+//
+// /**
+//  * @function
+//  * @param {Object} DateTimeEntity model object
+//  * @return {boolean} true if date has been postponed
+//  */
+// export const isPostponed = ( DateTimeEntity ) => {
+// 	assertDateTimeEntity( DateTimeEntity );
+// 	return DateTimeEntity.postponed;
+// };
```

Comment:
Maybe if leaving in commented code it'd be good to include an explanation for _why_ its being left?  Otherwise, it'd be better to just remove it all together.

Reply:
sure, will fix